### PR TITLE
Fix/icon

### PR DESCRIPTION
**Pull request by Crash--**
- Use PublicLayout for LightFileViewer
- Add `isFile` attribute to know if we are rendering a file or a folder.
- Added some proptypes 
- Changed Photos public view (buttons order / padding)

### CODE REVIEW

File: src/drive/web/modules/public/LightFileViewer.jsx [hunk 1]
```diff
@@ -15,4 +17,8 @@ const LightFileViewer = ({ files }) => (
   </div>
 )
 
+LightFileViewer.propTypes = {
+  files: PropTypes.array.isRequired,
```
Comment (cedricmessiant) [inline src/drive/web/modules/public/LightFileViewer.jsx]: could you use `arrayOf` so that we know exactly what do the files look like here?

File: src/drive/targets/public/index.jsx [hunk 2]
```diff
@@ -106,7 +107,6 @@ const init = async () => {
               <Redirect from="/*" to={`files/${id}`} />
             </Router>
           )}
-          <IconSprite />
```
Comment (cedricmessiant) [inline src/drive/targets/public/index.jsx]: why did you remove `IconSprite`? `PublicLayout` provides the icons?

File: src/drive/web/modules/public/PublicToolbar.jsx [hunk 3]
```diff
@@ -147,26 +165,34 @@ class PublicToolbar extends React.Component {
...
       )
     }
   }
 }
-
+PublicToolbar.propTypes = {
+  isFile: PropTypes.bool.isRequired,
+  renderInBar: PropTypes.bool,
+  breakpoints: PropTypes.object.isRequired,
+  files: PropTypes.array.isRequired
```
Comment (cedricmessiant) [inline src/drive/web/modules/public/PublicToolbar.jsx]: same here and same for the shape of the `breakpoints` object above

File: src/drive/targets/public/index.jsx (hunk 2 above)
Comment (Crash--) [reply src/drive/targets/public/index.jsx]: Exactly! https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/public/PublicLayout.jsx#L10

File: src/drive/web/modules/public/PublicToolbar.jsx (hunk 3 above)
Comment (Crash--) [reply src/drive/web/modules/public/PublicToolbar.jsx]: When cozy-ui will expose its own propTypes. Currently it's not the case and I don't want to put that kind of work in this PR.

File: src/drive/web/modules/public/LightFileViewer.jsx (hunk 1 above)
Comment (Crash--) [reply src/drive/web/modules/public/LightFileViewer.jsx]: Since `files` is something coming from the API, it can have a lot of different shapes. 

Is it possible to only check for the presence of one or two keys ?

**Comment by Crash--** [comment]
[PARTAGE - icones vues publiques (suite)](https://trello.com/c/Rs1koHfT/1560-partage-icones-vues-publiques-suite)
